features/admin.core.v1/models/config.ts

```diff
@@ -61,6 +61,13 @@ interface ConnectionConfigInterface extends FeatureAccessConfigInterface {
     templates?: Record<string, any>[];
 }
 
+export interface ApplicationsFeatureAccessConfigInterface extends FeatureAccessConfigInterface {
```

Comment:
Since we now take the feature status from `disabledFeatures` array, we do not need this interface anymore, do we?